MapboxDirections/MBLaneIndicationComponent.swift

```diff
@@ -0,0 +1,50 @@
+/**
+A component that represents a lane  representation of an instruction.
+ */
+@objc(MBLaneIndicationComponent)
+open class LaneIndicationComponent: NSObject, ComponentRepresentable {
+
+    /**
+     An array indicating which directions you can go from a lane (left, right, or straight).
+     
+     If the value is `[LaneIndication.left", LaneIndication.straightAhead]`, the driver can go left or straight ahead from that lane. This is only set when the `component` is a `lane`.
+     */
+    @objc public var indications: LaneIndication
+    
+    /**
+     The boolean that indicates whether the component is a lane and can be used to complete the upcoming maneuver.
```

Comment:
I think since this is a LaneIndicationComponent, we already know it's a lane. I think this is more about whether the lane can be used to complete the maneuver.